tests/visualization_tests/matplotlib_tests/test_contour.py

```diff
@@ -235,3 +238,53 @@ def test_generate_contour_plot_for_few_observations(params: List[str]) -> None:
     study = prepare_study_with_trials(less_than_two=True)
     figure = plot_contour(study, params)
     assert not figure.has_data()
+
+
+def all_equal(iterable: Iterable) -> bool:
+    """Returns True if all the elements are equal to each other"""
+    return len(set(iterable)) <= 1
```

Comment:
```suggestion
    return len(set(iterable)) == 1
```

I think `len(set(iterable))` returning `0` is not the expected scenario so it looks safer to check if the value is exactly one. Please let me know if you have the intention I missed.

Reply:
Agree!